packages/grid-pro/test/edit-column-renderer.test.js

```diff
@@ -289,4 +289,119 @@ describe('edit column renderer', () => {
       expect(grid.items[0].name).to.equal('New');
     });
   });
+
+  describe('programatically enter edit mode', () => {
+    let grid;
+
+    beforeEach(() => {
+      grid = fixtureSync(`
+        <vaadin-grid-pro>
+          <vaadin-grid-pro-edit-column path="name" id="name"></vaadin-grid-pro-edit-column>
+          <vaadin-grid-pro-column path="age"></vaadin-grid-pro-column>
+          <vaadin-grid-column>
+            <template>[[item.name]]</template>
```

Comment:
In case the column isn't essential for the test, it can be removed. If it is, see the documentation for how custom content is provided for grid column cells (`renderer` API)